components/ui/tab.tsx

```diff
@@ -25,7 +25,7 @@ export const TabPanels = (props: ExtractProps<typeof HeadlessTab.Panels>) => {
 export const TabList = (props: ExtractProps<typeof HeadlessTab.List>) => {
   return (
     <HeadlessTab.List
-      className="flex flex-row items-center gap-2 rounded-lg border border-gray-100 bg-gray-50"
+      className="flex flex-row items-stretch gap-2 rounded-lg border border-gray-100 bg-gray-50"
```

Comment:
This fixes some ui inconsistency I've noticed

Before
<img width="950" alt="image" src="https://user-images.githubusercontent.com/11313640/229052055-e854d320-ecca-4ae8-b2f3-9c6515d0946a.png">


After
<img width="940" alt="image" src="https://user-images.githubusercontent.com/11313640/229052159-9bb4c8c4-1576-4d75-9e93-734b0f543413.png">